docs/products/m3db/howto/write-go.rst

```diff
@@ -0,0 +1,42 @@
+Write Data to M3DB with Go
+--------------------------
+
+This example writes some data to an M3DB service from Go, making use of the Prometheus write features.
+
+Variables
+'''''''''
+
+These are the placeholders you will need to replace in the code sample:
+
+==================      ==========================================================
+Variable                Description
+==================      ==========================================================
+``PROM_WRITE_URL``      URL for Prometheus writes, from the service overview page
+==================      ==========================================================
+
+Pre-requisites
+''''''''''''''
+
+For this example you will need:
+
+1. The Prometheus client for Go::
+
+    go get -u github.com/m3db/prometheus_remote_client_golang/promremote
+
+ 
+
+Code
+''''
+
+Add the following to ``main.go`` and replace the placeholder with the Prometheus(write) URL:
```

Comment:
"replace the placeholder with the Prometheus(write) URL" -> "replace the placeholder with the Prometheus(write) URL you can find in the M3DB service overview"

Reply:
Hmm, it says that in the previous section. Maybe I just should say "placeholders"